web/styles/app_variables.css

```diff
@@ -767,6 +771,10 @@
     --color-message-formatting-controls-container: hsl(232deg 30% 96%);
     --color-message-content-container-border: hsl(0deg 0% 0% / 10%);
     --color-message-content-container-border-focus: hsl(0deg 0% 57%);
+    --color-message-content-container-border-over-limit: hsl(0deg 76% 65%);
+    --color-message-content-container-border-approaching-limit: hsl(
+        038deg 70% 50%
```

Comment:
Why is this "038deg" and not "38deg"?

Reply:
oh sorry, its a typo. Might have occured while pasting the hsl code.